### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by b1zeyofficial** [diamond-gold/DummyItemsBlocks#15] [Q&A] [answered]
Will the new version of PMMP 5.8.2 be supported? It would be easier if DummyItemsBlocks supported the new version of PMMP

**Comment by b1zeyofficial** [comment]
If you look at duplicate #[12](https://github.com/diamond-gold/DummyItemsBlocks/issues/12), the user claims that when you join the server 1.20.40 it crashes, I can assume that the block error may be the following new blocks added to pmmp
Implemented the following blocks:
Amethyst
Amethyst Cluster
Chiseled Bookshelf
Crimson Roots
Pitcher Crop
Pitcher Plant
Torchflower
Torchflower Crop
Warped Roots

**Reply by diamond-gold** [reply]
It is a client bug, update your minecraft to 1.20.41

**Comment by diamond-gold** [comment, marked as answer]
It already supports 5.8.2
